<?xml version="1.0"?>

<launch>

	<!-- Specify robot dimensions as parameters -->
	<rosparam file="$(find robo_mower)/config/robot_common_params.yaml" command="load" />

	<!-- rosserial node to arduino -->
  <node pkg="rosserial_python" type="serial_node.py" name="rosserial" output="screen">
		<param name="port" value="/dev/teensy35" />
		<param name="baud" value="115200" />
	</node>

	<!-- odom node subscribes to /cmd_vel, publishes /odom-->
  <node pkg="robo_mower" type="wheel_odometry" name="wheel_odometry" output="screen">
		<param name="output_frequency" value="30.0" />
	</node>

</launch>
